src/components/filters/MinMaxFilter/index.tsx

```diff
@@ -87,15 +88,27 @@ export const MinMaxFilter: React.FC<Props> = (props) => {
     emptyValues,
   ])
 
+  const validateBounds = React.useCallback(() => {
+    if (min) {
+      setMin(Math.trunc(min))
+      if (min < MIN_OUTCOMES_ALLOWED) setMin(MIN_OUTCOMES_ALLOWED)
+    }
+
+    if (max) {
+      setMax(Math.trunc(max))
+      if (max > MAX_OUTCOMES_ALLOWED) setMax(MAX_OUTCOMES_ALLOWED)
+    }
+  }, [max, min])
```

Comment:
For this logic we should take into account that setState operations are batched, so if max was `5`  and changes to `2.5` the callback is executed in this way

```
setMin(Math.trunc(2.5))
if(2.5 < MIN_OUTCOMES_ALLOWED) setMin(MIN_OUTCOMES_ALLOWED)
``` 
I guess this run again, but maybe is clearer if you only make one set
```
const max = Math.trunc(max) // or maxTruncated
setMax(max > MAX_OUTCOMES_ALLOWED ? MAX_OUTCOMES_ALLOWED : max)
```

Anyways it works in the way it is right now.

Reply:
Yeah, I like your solution more. Problem is: `min` or `more` could be `null` so I end up with something like this to make the compiler happy:

```
const minTrunc = min !== null ? Math.trunc(min) : null
const maxTrunc = max !== null ? Math.trunc(max) : null

setMin(minTrunc !== null && minTrunc < MIN_OUTCOMES_ALLOWED ? MIN_OUTCOMES_ALLOWED : minTrunc)
setMax(maxTrunc !== null && maxTrunc > MAX_OUTCOMES_ALLOWED ? MAX_OUTCOMES_ALLOWED : maxTrunc)
```

Not sure if there's a simpler solution.

Reply:
You could use a short-circuit &&, and do something like `min && Math.trunc(min)`. 0 is considered falsy but 0 truncated is 0, in my opinion, it's ok in the last version.